go/vt/vttls/vttls.go

```diff
@@ -23,10 +23,37 @@ import (
 	"io/ioutil"
 )
 
+// Updated list of acceptable cipher suits to address
+// Fixed upstream in https://github.com/golang/go/issues/13385
+// This removed CBC mode ciphers that are suseptiable to Lucky13 style attacks
+func newTLSConfig() *tls.Config {
+	return &tls.Config{
+		// the MySQL client on the CI server does not work with tls1.2
+		// Newer versions of the client do work
+		// TODO: Update the CI server to use a newer version of the mysql client
+		//MinVersion: tls.VersionTLS12,
+
+		// Default ordering taken from
+		// go 1.11 crypto/tls/cipher_suites.go
+		CipherSuites: []uint16{
+			tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
+			tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
+			tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
```

Comment:
When we get to TLS 1.3 we might want to add the CCM suites. But omitting them is safe.
```
      CipherSuite.TLS_AES_128_CCM_SHA256,
      CipherSuite.TLS_AES_256_CCM_8_SHA256,
```

Reply:
When https://github.com/golang/go/issues/9671 lands we need to update this to include the new ciphers